slave-prod-centos7/Dockerfile

```diff
@@ -6,8 +6,9 @@ MAINTAINER Adam Saleh <asaleh@redhat.com>
 USER root
 
 RUN curl -Lk -o /etc/yum.repos.d/rcm-tools.repo http://download-ipv4.eng.brq.redhat.com/rel-eng/RCMTOOLS/rcm-tools-rhel-7-server.repo
+RUN curl --silent --location https://rpm.nodesource.com/setup_6.x | bash -
 
-RUN yum update -y && yum install -y krb5-workstation rhpkg
+RUN yum update -y && yum install -y krb5-workstation rhpkg  ansible python2-simplejson python2-lxml nodejs krb5-workstation
```

Comment:
You've got `krb5-workstation` there already, so probably don't need to add it again